source/includes/job-board/_applications.md

```diff
@@ -59,12 +59,19 @@ curl -X POST \
   -F "educations[][start_date][year]=2012" \
   -F "educations[][end_date][month]=5" \
   -F "educations[][end_date][year]=2016" \
+  -F 'employments[][company_name]=Business Co' \
+  -F 'employments[][title]=Sales Manager' \
+  -F 'employments[][start_date][month]=1' \
+  -F 'employments[][start_date][year]=2016' \
+  -F 'employments[][end_date][month]=2' \
+  -F 'employments[][end_date][year]=2018' \
+  -F 'employments[][current]=false'
```

Comment:
Can we use double quotes for these employments fields just for the sake of unified styling with the other requests around it?